libs/vkd3d-common/profiling.c

```diff
@@ -192,4 +226,34 @@ void vkd3d_profiling_notify_work(unsigned int index,
     spinlock_release(lock);
 }
 
+#ifdef TRACY_ENABLE
+
+void ___vkd3d_set_thread_name( const char* name )
```

Comment:
Formatting is all over the place here. Do we need triple underscores? (reserved identifier)

Reply:
I did that strictly to follow Tracys naming scheme.